src/database/models/exile.py

```diff
@@ -1,9 +1,17 @@
 class Exile(object):
```

Comment:
nit, and you can probably ignore this, but I'd like to move our DB models away from extending `object` and instead extend pydantic's `BaseModel` like the User is set up. I don't think you should do this here, just felt like commenting because you're updating here